_includes/articles/veriphi-segwit-batching.md

```diff
@@ -38,24 +38,21 @@ code used [here][veriphi segwit repo].
 
 __Major takeaways__
 
-Assuming a bitcoin price of $9,250.00 (correct at the time of publication):
+Assuming a bitcoin price of $9,250 (correct at the time the report was published):
 
-- From January 2012 to June 2020 (until block 637,090th) 211,266.95 bitcoins
-  were paid in fees to miners. This amounts to a total of around $1,954,219,287
+- From January 2012 to June 2020 (until block 637,090) 211 thousand BTC
+  were paid in fees to miners. This amounts to a total of around $2 billion
   USD.
-- Total savings during that period amounted to 57,817.69 bitcoins representing
-  an amount of $534,813,632.5 USD. This represents a percentage of around 27.36%
+- Total savings during that period amounted to almost 58 thousand BTC representing
+  an amount of about $530 million USD. This represents a percentage of around 27.36%
   over the grand total of transaction fees paid.
-- Over 21,131.97 bitcoins could have been saved by Bitcoin users if they would
-  have all been using transaction batching. 190,134.98 bitcoins could have been
-  paid in fees instead of the 211,941.32 bitcoins, which represents savings of
-  9.97%. The 21,131.97 bitcoins saved represents a staggering amount of
-  $195,470,722 USD.
-- From August 24th 2017 to the 30th of June 2020, 36,685.72 bitcoins could have
-  been saved by Bitcoin users if they would have all been using segwit Native
-  (Bech32). Fees would have amounted to 59,848.61 bitcoins, down from 96,534.33
-  bitcoins actually paid in fees, which is 38.00% in savings. The 36,685.72
-  bitcoins saved represent $339,342,910 USD.
+- Over 21 thousand BTC could have been saved by Bitcoin users if they
+  would have all been using transaction batching, a savings of 9.97% or
+  almost $200 million USD.
+- From 24 August 2017 to 30 June 2020, about 37 thousand BTC could have
+  been saved by Bitcoin users if they would have all been using segwit native
+  (bech32), down from over 97 thousand BTC
+  actually paid in fees, which is 38.00% in savings or about $340 million USD.
```

Comment:
ACK on this commit 6067ddfac22b7dac. If you retouch, consider maybe changing 9.97% to 10% and 38.00% to 38%.